db/migrate/20241210203947_remove_searchgov_urls_fetch_required_index.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveSearchgovUrlsFetchRequiredIndex < ActiveRecord::Migration[7.1]
+  def change
+    remove_index :searchgov_urls, name: :searchgov_urls_fetch_required, if_exists: true
```

Comment:
This indexes need to be added back https://github.com/GSA/search-gov/blob/8bd58066cb65d9429e5e47d23b7c4e8f103c95f9/db/migrate/20230926194650_remove_unused_searchgov_urls_indexes.rb